scripts/docs.py

```diff
@@ -0,0 +1,307 @@
+import json
+import logging
+import os
+import re
+import shutil
+import subprocess
+from functools import lru_cache
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+from importlib import metadata
+from multiprocessing import Pool
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union
+
+import mkdocs.commands.build
+import mkdocs.commands.serve
+import mkdocs.config
+import mkdocs.utils
+import typer
+import yaml
+from jinja2 import Template
+
+logging.basicConfig(level=logging.INFO)
+
+app = typer.Typer()
+
+mkdocs_name = "mkdocs.yml"
+
```

Comment:
what is this trying to do?
maybe you can mention this on your PR
or add a doc on top of file, explaining what it does.
i thik it may also be related to fastapi use case (i didn't read the whole thing)